### PR TITLE
no log delay

### PR DESCRIPTION
**Pull request by Schparky**
Configure Yii logging to immediately output log messages, rather than cache them up. Prevents messages from being lost if the container crashes (e.g. out of memory).

### CODE REVIEW

File: application/frontend/config/main.php [hunk 1]
```diff
@@ -91,6 +91,7 @@
             ]
         ],
         'urlManager' => [
+            'cache' => null,
```
Comment (mtompset) [inline application/frontend/config/main.php:94]: "Set this property to false or null if you do not want to cache the URL rules."
https://www.yiiframework.com/doc/api/2.0/yii-web-urlmanager

File: application/common/config/main.php [hunk 2]
```diff
@@ -101,6 +101,7 @@
                     'logVars' => [],
                     'categories' => ['application'],
                     'prefix' => $logPrefix,
+                    'exportInterval' => 1,
```
Comment (mtompset) [inline application/common/config/main.php:104]: "How many messages should be accumulated before they are exported. Defaults to 1000."
https://www.yiiframework.com/doc/api/2.0/yii-log-target#$exportInterval-detail

File: application/frontend/config/main.php (hunk 1 above)
Comment (forevermatt) [reply application/frontend/config/main.php:94]: What does not-caching URL rules gain us?

Comment (Schparky) [reply application/frontend/config/main.php:94]: I believe this is specifying the default, which we have been using implicitly. Specifying it here silences a warning message.

**Pull request merged**